Description: fix POD to get a whatis entry after pod2man
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-04-13

--- a/README.pod
+++ b/README.pod
@@ -1,4 +1,6 @@
-=head1 DBD::mysql - database driver for Perl
+=head1 NAME
+
+DBD::mysql - database driver for Perl
 
 This is the Perl L<DBI> driver for access to MySQL databases.
 
